trino/base/hive-metastore-dc.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: hive
+    hive: metastore
+  name: hive-metastore
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: hive
+      hive: metastore
+  serviceName: hive-metastore
+  template:
+    metadata:
+      labels:
+        app: hive
+        hive: metastore
+    spec:
+      containers:
+      - name: metastore
+        args:
+          - /opt/hive/bin/hive
+          - --service
+          - metastore
+        command:
+          - /hive-scripts/entrypoint.sh
+        env:
+          - name: HIVE_LOGLEVEL
+            value: INFO
+          - name: AWS_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                key: AWS_ACCESS_KEY_ID
+                name: aws-secret
+          - name: AWS_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                key: AWS_SECRET_ACCESS_KEY
+                name: aws-secret
+          - name: S3_BUCKET_NAME
+            valueFrom:
+              secretKeyRef:
+                key: BUCKET
+                name: aws-secret
+          - name: S3_DATA_DIR
+            value: data
+          - name: DATABASE_USER
+            valueFrom:
+              secretKeyRef:
+                key: database-user
+                name: trino-db-secret
+          - name: DATABASE_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: database-password
+                name: trino-db-secret
+          - name: DATABASE_HOST
+            value: trino-db
+          - name: DATABASE_PORT
+            value: 5432
+          - name: DATABASE_NAME
+            value: trino
+          - name: DATABASE_SSLMODE
+            value: accept
```

Comment:
Checking PostgreSQL documentation I can't find this value. Valid options are: `disable`, `allow`, `prefer`, `require`, `verify-ca` and `verify-full`. Source: https://www.postgresql.org/docs/9.1/libpq-ssl.html

Reply:
Typo. Fixed.